docs/content/dagster-cloud/account/setting-up-alerts.mdx

```diff
@@ -5,3 +5,226 @@ noindex: True
 ---
 
 # Setting up alerts in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+In this guide, we'll walk you through configuring job alerts in Dagster Cloud.
+
+---
+
+## Understanding alert policies
+
+Alert policies define which jobs will trigger an alert, the conditions under which an alert will be sent, and how the alert will be sent.
+
+An alert policy includes a set of configured tags. If an alert policy has no configured tags, all jobs will be eligible for that alert. Otherwise, only jobs that contain all the tags for a given alert policy are eligible for that alert.
+
+Currently, Slack and email notifications are supported.
+
+---
+
+## Managing alert policies in Dagster Cloud
+
+<Note>
+  <strong>Organization Admin</strong> or <strong>Admin</strong> permissions are
+  required to manage alerts in Dagster Cloud.
+</Note>
+
+- [Creating alert policies](#creating-alert-policies)
+- [Editing alert policies](#editing-alert-policies)
+- [Enabling and disabling alert policies](#enabling-and-disabling-alert-policies)
+- [Deleting alert policies](#deleting-alert-policies)
+
+### Creating alert policies
+
+1. Sign in to your Dagster Cloud account.
+
+2. Click the **user menu (your icon) > Cloud Settings**.
+
+3. Click the **Alerts** tab.
+
+4. Click **+ Create alert policy**.
+
+5. In the **Create alert policy** window, fill in the following:
+
+   - **Alert policy name** - Enter a name for the alert policy. For example, `slack_urgent_failure`
+   - **Description** - Enter a description for the alert policy
+   - **Tags** - Add tag(s) for the alert policy. Jobs with these tags will trigger the alert. For example: `level:critical` or `team:sales`
+   - **Events** - Select whether the alert should trigger on job success, failure, or both
```

Comment:
we do plan to add more events in the future (hopefully soon), can always edit this when we do